core/consensus/babe/primitives/src/lib.rs

```diff
@@ -116,7 +134,7 @@ decl_runtime_apis! {
 		/// Dynamic configuration may be supported in the future.
 		fn startup_data() -> BabeConfiguration;
 
-		/// Get the current authorites for Babe.
-		fn authorities() -> Vec<AuthorityId>;
+		/// Get the current epoch data for Babe.
+		fn authorities() -> Epoch;
```

Comment:
Can't we rename the method to `epoch()`?

Reply:
We can, and should.  Will fix.